docs/src/rules/constructor-super.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: constructor-super
 rule_type: problem
+handled_by_typescript:
+- 2335
+- 2377
```

Comment:
Just noting that these version numbers aren't guaranteed to exist version-to-version: https://github.com/microsoft/TypeScript/issues/19139[^1]

[^1]: I think there's an issue floating around somewhere in the TypeScript issue tracker that more explicitly states the errors can change version-to-version... but I couldn't find it after 5 minutes. 🤷